lstm/bmi_lstm.py

```diff
@@ -448,15 +448,15 @@ def get_output_var_names(self) -> tuple[str, ...]:  # type: ignore
 
     def get_var_grid(self, name: str) -> int:
         # Note: all vars have grid 0 but check if its in names list first
-        if name in StateFacade(self._outputs, self._dynamic_inputs):
-            return 0
+        # raises KeyError on failure
+        first_containing(name, self._outputs, self._dynamic_inputs)
```

Comment:
This doesn't return now.

And we have conflicting exceptions....the `RuntimeError` indicates an "unknown name" which is effectively the same as the `KeyError` but with a different exception type and context.

Reply:
Great catch! Just going too fast. Now just returns `0` if a `KeyError` is not found.